packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -985,25 +988,35 @@ const runGenerator = async (
         processAppExtensions(selectedAppExtensions, extractAppExtensions, appExtensionsDir)
     }
 
-    // Add selected Application Extensions to devDependencies
-    const appExtensionDeps = selectedAppExtensions.reduce((acc, appExtensionName) => {
-        // Find the corresponding Application Extension details
-        const appExtensionDetails = context?.availableAppExtensions?.find(
-            (ext) => ext.value === `${appExtensionName}@latest`
-        )
-        const version = appExtensionDetails ? appExtensionDetails.version : 'latest'
-
-        acc[appExtensionName] = extractAppExtensions
-            ? `file:./app/application-extensions/${appExtensionName}`
-            : version
-        return acc
-    }, {})
-
-    updatePackageJson(p.resolve(outputDir, 'package.json'), {
+    // Prepare updates for package.json
+    const pkgUpdates = {
         name: getSlugifiedProjectName(context.answers.project.name || context.preset.id),
         version: GENERATED_PROJECT_VERSION,
-        devDependencies: appExtensionDeps
-    })
+        scripts: {
+            start: `npm --prefix ./${LOCAL_DEV_PROJECT_DIR} start`,
+            'start:inspect': `npm --prefix ./${LOCAL_DEV_PROJECT_DIR} run start:inspect`
+        },
+        // Conditionally add workspaces for extractAppExtensions
+        ...(extractAppExtensions && {
+            workspaces: ['app/application-extensions/*']
```

Comment:
Having a constant for this would be nice do we can change the name in the future to `/extensions` or `/_extensions`